include/ForceColl/Contact.h

```diff
@@ -150,6 +156,8 @@ class EmptyContact : public Contact
   {
     return "Empty";
   }
+
+  inline virtual void updateGlobalVertices(const sva::PTransformd & pose) {}
```

Comment:
Add `override` to the end of the declaration. Do the same for all override methods in other classes.

Reply:
Fixed in 5c962994a51049355a95d8ab509505676d082a25